lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -15,6 +15,10 @@ include_once 'dosomething_user.theme.inc';
  */
 function dosomething_user_preprocess_page(&$vars) {
   $school_api_endpoint = variable_get('dosomething_user_school_api_endpoint', 'http://lofischools.herokuapp.com/search');
+  if ($_SESSION['log_login']) {
```

Comment:
I think @mshmsh5000 @sheydari-ds or @blisteringherb might have some feelings on how this works with fastly